libenkf/include/ert/enkf/enkf_main.h

```diff
@@ -139,7 +139,7 @@ extern "C" {
   void                          enkf_main_load_obs( enkf_main_type * enkf_main , const char * obs_config_file , bool clear_existing);
   enkf_obs_type               * enkf_main_get_obs(const enkf_main_type * );
   bool                          enkf_main_have_obs( const enkf_main_type * enkf_main );
-  analysis_config_type        * enkf_main_get_analysis_config(const enkf_main_type * );
+  const analysis_config_type  * enkf_main_get_analysis_config(const enkf_main_type * );
```

Comment:
This `const` is tempting - but the `analysis_config` object will be mutated in the tui: https://github.com/Statoil/ert/blob/master/libenkf/applications/ert_tui/enkf_tui_analysis.c#L76 - but I guess the Good&trade; solution is to do `const` return as in this PR, and just remove the offending functionality/menu option from the TUI.